stdlib/map.gr

```diff
@@ -278,24 +278,28 @@ export let fromList = (list) => {
 }
 
 let setInArray = (array) => {
-  @disableGC
```

Comment:
Is this truly okay? The array here is uninitialized, so won't it be dangerous to set into the array, since the old item will be decRef'd? I know that decRef handles null just fine, but random values probably will cause problems.